salt/utils/event.py

```diff
@@ -586,6 +591,9 @@ def __del__(self):
             self.destroy()
         except Exception as ex:
             log.debug(ex)
+            # as this method can be called when logging is already teared down
+            # just try to stdout it
+            traceback.print_exc(file=sys.stderr)
```

Comment:
Just saw your comment above this call...

I think we can know if logging has been teared down...